scripts/spack/configs/docker/ubuntu18/packages.yaml

```
@@ -101,8 +101,11 @@ packages:
   hdf5:
     variants: ~shared~mpi
   # do shared mfem to allow PIC flag in mfem
+  hypre:
+    version: [2.20.0]
```

I think the comment should stay on top of the mfem section:

```suggestion
  hypre:
    version: [2.20.0]
  # do shared mfem to allow PIC flag in mfem
```